### PR TITLE
Implement about block

### PR DESCRIPTION
**Pull request by sophieH29**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -14,9 +14,11 @@
     "paper-icon-button": "PolymerElements/paper-icon-button#^2.0.0",
     "iron-image": "PolymerElements/iron-image#^2.0.0",
     "paper-tabs": "PolymerElements/paper-tabs#^2.0",
-    "polymer-redux": "tur-nr/polymer-redux#polymer-2",
+    "polymer-redux": "tur-nr/polymer-redux",
```
Comment (ozasadnyy) [inline bower.json]: `"polymer-redux": "tur-nr/polymer-redux#^1.0.1"`

File: data/resources.json [hunk 2]
```diff
@@ -74,5 +74,41 @@
...
+  "codeOfConduct": "Code of Conduct",
+  "aboutBlock": {
+    "title": {
+      "partA": "What you need to know,",
+      "partB": "before you ask."
+    },
+    "descriptionBlocks": [
+      {
+        "description": "GDG DevFest Ukraine brings together the world class experts in Android, Web and Cloud technologies to Lviv for 2 days of sessions, workshops and showcases.",
+        "ctaLabel": "Explore featured session",
```
Comment (ozasadnyy) [inline data/resources.json]: I would rather create an object:
```
callToAction: {
  label: '',
  link: ''
}
```

File: src/elements/about-block.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                margin: 48px 40px;
+            }
+
+            .description-block,
+            .statistics-block {
+                padding: 16px;
+            }
+
+            .statistics-block .row-1 {
+                margin-bottom: 16px;
```
Comment (ozasadnyy) [inline src/elements/about-block.html]: Why not `margin-bottom: 13px` and remove next css block?

File: src/elements/about-block.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                    margin: 0 48px;
+                }
+             }
+
+        </style>
+
+        <div class="container" flex layout horizontal center wrap-reverse>
+
+            <div class="description-block" flex>
+                <h2 class="title">{$ aboutBlock.title.partA $}</br>{$ aboutBlock.title.partB $}</h3>
```
Comment (ozasadnyy) [inline src/elements/about-block.html]: use `max-width` instead of a breaking sentence manually

File: src/hoverboard-app.html [hunk 5]
```diff
@@ -38,6 +39,13 @@
         min-height: 100vh;
       }
 
+      /* TODO: Remove after hero-block element implemeted */
```
Comment (ozasadnyy) [inline src/hoverboard-app.html]: forgot to remove

File: src/hoverboard-app.html [hunk 6]
```diff
@@ -177,8 +185,14 @@ <h3 class="location">{$ location.short $}</h3>
             </paper-tabs>
           </app-toolbar>
         </app-header>
-
-        <h1>{{routeData.page}}</h1>
+        
+        <!--TODO: Replace with hero-block element-->
```
Comment (ozasadnyy) [inline src/hoverboard-app.html]: same here

File: data/resources.json (hunk 2 above)
Comment (sophieH29) [inline data/resources.json]: agree

File: src/elements/about-block.html (hunk 3 above)
Comment (sophieH29) [inline src/elements/about-block.html]: why 13? :) We have 2 horizontal blocks which have to have 32px between them. 
I could put 32px to bottom of one elem, but that's not good practice as this element would keep too much space around it, as the another block wouldn't have it

File: src/elements/about-block.html (hunk 4 above)
Comment (sophieH29) [inline src/elements/about-block.html]: max-width wouldn't work when the width becomes smaller. Better just to set width.

File: src/elements/about-block.html (hunk 3 above)
Comment (ozasadnyy) [inline src/elements/about-block.html]: Yeah, 32. It was a mistake. You can use `:not(:last-of-type)` as alternative solution

File: src/elements/shared-animations.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+                to {
+                    transform: translate3d(0, 0, 0);
+                }
+            }
+
+            @keyframes fadeInTracksCount {
+                from {
+                    opacity: 0;
+                    transform: translate3d(177px, -170px, 0px);
```
Comment (ozasadnyy) [inline src/elements/shared-animations.html]: Why 177? :)

File: src/elements/about-block.html (hunk 3 above)
Comment (sophieH29) [inline src/elements/about-block.html]: I would like to leave this solution

File: src/elements/shared-animations.html (hunk 7 above)
Comment (sophieH29) [inline src/elements/shared-animations.html]: Because it's a distance I choose to animate the element :)

File: src/elements/about-block.html (hunk 3 above)
Comment (ozasadnyy) [inline src/elements/about-block.html]: I don't like a solution in naming `row-1`, `row-2`. I recommend using pseudo classes, it's more abstract

File: src/elements/shared-animations.html (hunk 7 above)
Comment (ozasadnyy) [inline src/elements/shared-animations.html]: please use rounded values, 177 looks weird)

**Pull request merged**